benches/bench_for_estimates.rs

```diff
@@ -0,0 +1,287 @@
+#![allow(bare_trait_objects)]
```

Comment:
We probably don't need these allow() statements anymore. Especially not the bare trait objects one. If the non_snake_case one helps write more clear benchmark names, then feel free to keep it.

Reply:
I don't get any warnings when I remove them, so no they are not needed any more.